src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(prevValue, value) {
+      return prevValue + value;
+    },
```

Comment:
both of arguments should have the same meaning
why one of them is prev value, and other one is value?
let's think more about names for variables